spec/features/api_can_serve_all_posts_spec.rb

```diff
@@ -0,0 +1,14 @@
+# localhost:3000/api/v1/posts
+
+require 'rails_helper'
+
+RSpec.feature "Posts API", type: :feature do
+  scenario "Visiting /api/v1/posts returns all posts" do
+
+    Post.create(message:"Hello!", slug:"HelloWorld")
+    visit '/api/v1/posts'
+    # expect(page).to have_content('"id": 1')
+    expect(page).to have_content('"message":"Hello!"')
+    expect(page).to have_content('"slug":"HelloWorld"')
+  end
+end
```

Comment:
Excellent test! is a post having a slug the functionality we want what does a slug do?